src/sidebar/services/annotations.js

```diff
@@ -116,6 +116,22 @@ export default function annotationsService(api, store) {
     });
   }
 
+  /**
+   * Delete an annotation via the API and update the store.
+   */
+  async function delete_(annotation) {
```

Comment:
The function is named `delete_` because you can't call a function or binding `delete`, although you _can_ use that name for a method.